account_financial_report/wizard/trial_balance_wizard.py

```diff
@@ -138,8 +140,10 @@ def onchange_company_id(self):
             res['domain']['account_ids'] += [
                 ('company_id', '=', self.company_id.id)]
             res['domain']['partner_ids'] += [
+                '&',
                 '|', ('company_id', '=', self.company_id.id),
-                ('company_id', '=', False)]
+                ('company_id', '=', False),
+                ('parent_id', '=', False)]
```

Comment:
The same about the domain. As this domain is repeated through a lot of places, I think it should be extracted to a method in the abstract model or similar.

Reply:
Done